client/src/services/textFormat.js

```diff
@@ -9,6 +9,12 @@ const formatDateToString = (date) => {
   return string.split(',')[0];
 };
 
+// Format Date type values to strings of the format "2023-01-01"
```

Comment:
Purely a comment: Should we look into our dates more closely? I feel like we have quite a few date formatting functions and I think it might be confusing one day when new people start working on this.

Reply:
Yeah we could talk about this. There are different formats coming in from the server (from our instances and Sisu instances) and also different formats are needed for the UI, so that's why there are many functions to format them.